src/commands/createFunctionApp/stacks/FunctionAppEOLWarningStep.ts

```diff
@@ -0,0 +1,31 @@
+/*---------------------------------------------------------------------------------------------
+*  Copyright (c) Microsoft Corporation. All rights reserved.
+*  Licensed under the MIT License. See License.md in the project root for license information.
+*--------------------------------------------------------------------------------------------*/
+import { AzureWizardPromptStep, DialogResponses, openUrl } from "@microsoft/vscode-azext-utils";
+import { MessageItem } from "vscode";
+import { funcVersionLink } from "../../../FuncVersion";
+import { localize } from "../../../localize";
+import { getWorkspaceSetting, updateGlobalSetting } from "../../../vsCodeConfig/settings";
+import { IFunctionAppWizardContext } from "../IFunctionAppWizardContext";
+
+export class FunctionAppEOLWarningStep extends AzureWizardPromptStep<IFunctionAppWizardContext> {
+    public async prompt(context: IFunctionAppWizardContext): Promise<void> {
+        const settingKey: string = 'endOfLifeWarning';
+        if (getWorkspaceSetting<boolean>(settingKey)) {
+            const message = localize('endOfLife', "The chosen runtime stack has an end of support deadline coming up." +
```

Comment:
Needs spaces after the periods

<img width="513" alt="image" src="https://user-images.githubusercontent.com/12476526/200429461-0e517b49-3ec6-4d3c-8b17-258a0307e5e7.png">

Reply:
And add a period for the last sentence